LSPDFR+/Menus.cs

```diff
@@ -665,35 +666,47 @@ public static void UpdateTrafficStopQuestioning()
         private static bool CourtsMenuPaused = false;
         private static void Process(object sender, GraphicsEventArgs e)
         {
-            if (Functions.IsPlayerPerformingPullover() && !_MenuPool.IsAnyMenuOpen() && EnhancedTrafficStop.EnhancedTrafficStopsEnabled)
+            try
             {
-                if (Vector3.Distance2D(Game.LocalPlayer.Character.Position, Functions.GetPulloverSuspect(Functions.GetCurrentPullover()).CurrentVehicle.Position) < TrafficStopMenuDistance + 0.1f)
+                if (Functions.IsPlayerPerformingPullover() && !_MenuPool.IsAnyMenuOpen() && EnhancedTrafficStop.EnhancedTrafficStopsEnabled)
                 {
-                    ExtensionNamespace.Extensions.DisEnableControls(false);
-                    //ExtensionNamespace.Extensions.DisableTrafficStopControls();
-
-                    if (ExtensionMethods.IsKeyDownComputerCheck(EnhancedTrafficStop.BringUpTrafficStopMenuKey) || Game.IsControllerButtonDown(EnhancedTrafficStop.BringUpTrafficStopMenuControllerButton))
+                    Ped pulloverSuspect = Functions.GetPulloverSuspect(Functions.GetCurrentPullover());
+                    if (pulloverSuspect &&
+                        pulloverSuspect.IsInAnyVehicle(false) &&
+                        Vector3.Distance2D(Game.LocalPlayer.Character.Position, pulloverSuspect.CurrentVehicle.Position) < TrafficStopMenuDistance + 0.1f
+                        )
                     {
-                        _MenuPool.ResetMenus(true, true);
-                        SeizeVehicleTicketCheckboxItem.Enabled = true;
-                        TicketMenu.ParentMenu = TrafficStopMenu;
-                        TicketMenu.Visible = false;
-                        foreach (UIMenu m in OffenceCategoryMenus)
+                        ExtensionNamespace.Extensions.DisEnableControls(false);
```

Comment:
This line of code was to disable the 0.3b GameControl from LSPDFR (effectively preventing you from opening their traffic stop menu). It seems they use a different system now so this is no longer an effective line of code and takes away a lot of performance unnecessarily. Worth commenting/removing